tests/integration/helpers/cluster.py

```diff
@@ -770,6 +797,13 @@ def schema_registry_auth_port(self):
         self._schema_registry_auth_port = self.port_pool.get_port()
         return self._schema_registry_auth_port
 
+    @property
+    def kafka_sasl_port(self):
+        if self._kafka_sasl_port:
+            return self._kafka_sasl_port
+        self._kafka_sasl_port = self.port_pool.get_port()
+        return self._kafka_sasl_port
```

Comment:
This looks redundant

Reply:
Fixed in https://github.com/ClickHouse/ClickHouse/pull/83581/commits/fd81d6c302afbfe8c9dd60c35ebca1f5bbc6b87b where I chose only the rabbitmq-related changes